PDSim/core/core.py

```diff
@@ -1493,7 +1495,7 @@ def attach_HDF5_annotations(self, fName):
         
         hf = h5py.File(fName,'a')
         
-        for k, v in attrs_dict.iteritems():
+        for k, v in attrs_dict.items():
```

Comment:
Does this work in both py2.7 and py3.6?  You might need to use six.iteritems(attrs_dict)

Reply:
Indeed, that does seem fine in both of py27 and py36.  A new one for me, I'll have to change my code

Reply:
I found out about that modification after I PR and I was not sure about 2.x and 3.x. But .items( ) seems working for both distributions. Locally, I am only running 3.6